packages/expo-cli/src/commands/client/index.js

```diff
@@ -77,16 +83,25 @@ export default program => {
       if (addUdid) {
         urlOpts.printQRCode(result.registrationUrl);
         log(
-          chalk.green(
-            `Open this link on iOS to register the device and install the client:\n\n${
-              result.registrationUrl
-            }`
-          )
+          'Open the following link on your iOS device (or scan the QR code) and follow the instructions to install the development profile:'
         );
         log.newLine();
+        log(chalk.green(`${result.registrationUrl}`));
+        log.newLine();
+        log(
+          "Nothing will be saved on your device, but this step is necessary to get your device's unique identifier (UDID)."
```

Comment:
This is the only message I’m not sure about. Why would saving something to the device be a concern / something we need to assure is not going to happen?

Reply:
sorry for letting this rot 😞 I think you're right, this message isn't necessary. I've gone ahead and removed it.